src/views/share.njk

```diff
@@ -0,0 +1,51 @@
+{% extends "layouts/base.njk" %}
```

Comment:
I wonder if we should be making use of extending page, rather than base? A bit clearer and if we make customisations we dont need to mess with base too much. Thoughts?

Reply:
Cheers, makes sense.
Changed to extend page.njk